test/e2e/handler/upgrade_test.go

```diff
@@ -23,6 +23,8 @@ var _ = Describe("NodeNetworkConfigurationPolicy upgrade", func() {
 				},
 				Spec: nmstate.NodeNetworkConfigurationPolicySpec{
 					DesiredState: linuxBrUp(bridge1),
+					NodeSelector: map[string]string{"node-role.kubernetes.io/worker": ""},
+					Parallel:     parallelRollout,
```

Comment:
This is not related to the PR, is it?

Reply:
Explaineed at the commit messages.

Reply:
I don't see this one being explained, am I just missing it?

Reply:
The test was being applied a master and workers but being cleaned up only at worked also it was not being executed in parallel at parallel lane.